SECURITY.md

```diff
@@ -0,0 +1,64 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+We take the security of Boston Open Source's website and community infrastructure seriously. If you discover a security vulnerability, please follow these steps:
+
+1. **DO NOT** create a public GitHub issue
+2. Report via one of these secure channels (in order of preference):
+   - Send a private message on [Zulip](https://osdc.zulipchat.com/#narrow/stream/406743-boston) to the maintainers
```

Comment:
The current maintainers are me and @orblivion. Should we list ourselves here?

Reply:
I see a 👍 from @iAnonymous3000. What do you think @orblivion?

Reply:
Yeah that's a public channel right? We'd basically be asking people to join Zulip if they haven't yet, find our accounts (which we'd have to lists here), and DM us there.

Reply:
To answer directly: I am okay with being listed here.

I suppose that for people who already have Zulip, it would be a convenient option. But then, I don't see anything like a profile URL or ID. Someone could be a fake Dan or Phil and copy our photos.

What about plain email? If you don't want to list it here, you could say "go to bostonopen.dev and find the contact address" since it is listed there.